dev/ProgressRing/ProgressRing.xaml

```diff
@@ -15,35 +14,26 @@
         <Setter Property="MinWidth" Value="16" />
         <Setter Property="IsTabStop" Value="False" />
         <Setter Property="Width" Value="32" />
+        <Setter Property="Maximum" Value="100" />
```

Comment:
>Maximum [](start = 26, length = 7)

What is the default?

Reply:
The default should be 100, as outlined in the specs [here](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/progress/ProgressRing.md).

Reply:
then why do we need to set it to 100?

Reply:
`Maximum` currently has a default of 1 as it is inherited from RangeBase (outlined [here](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.primitives.rangebase.maximum?view=netcore-3.1)).